tox.ini

```diff
@@ -58,10 +59,18 @@ commands =
 deps =
     {[testenv]deps}
     -r{toxinidir}/requirements/dev.txt
-use_develop = true
+commands =
+    python -m pytest -ra --showlocals --csv={toxworkdir}/{envname}.csv {posargs:tests/integration/{[testenv]test_dir}}
+
+
+[testenv:unittest-all-{py38,py39,py310}]
+deps =
+    {[testenv]deps}
+    -r{toxinidir}/requirements/dev.txt
+extras = full
```

Comment:
It's just a question. `extras` added below has `all` value and here is `full`. Doesn't it matter?